runtime/src/migrations/initialise_ti.rs

```diff
@@ -0,0 +1,46 @@
+use frame_support::traits::{fungible, OnRuntimeUpgrade};
+
+use crate::*;
+
+pub struct Migration;
+
+impl OnRuntimeUpgrade for Migration {
```

Comment:
What is the advantage of having this migration in the runtime? I find it confusing that we have some migrations in the pallet , and some in the runtime. Can we move it to the subtensor module?

Reply:
They should be in the runtime actually, it allows pre/post checks and better integration with try-runtime-cli.

Reply:
I moved the migration definition to the pallet.

It's best practice for the migrations to be passed via the runtime so that the try-runtime pre/post hooks execute correctly.